source/lmp/tests/test_lammps_spin_dpa3_pt2.py

```
@@ -0,0 +1,304 @@
+            str(pb_file_mpi.resolve()),
+            out_path,
+        ]
+        if processors is not None:
+            argv.extend(["--processors", processors])
+        elif nprocs == 1:
+            argv.extend(["--processors", "1 1 1"])
+        if extra_args:
+            argv.extend(extra_args)
+        if runner_args:
+            argv.extend(runner_args)
+        sp.check_call(argv)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)